Corona-Warn-App/build.gradle

```diff
@@ -367,7 +367,7 @@ dependencies {
     // QR
     implementation('com.journeyapps:zxing-android-embedded:4.1.0') { transitive = false }
     // noinspection GradleDependency - needed for SDK 23 compatibility, in combination with com.journeyapps:zxing-android-embedded:4.1.0
-    implementation 'com.google.zxing:core:3.3.0'
+    implementation 'com.google.zxing:core:3.4.1'
```

Comment:
Please revert ? newer versions has compatibility issues with Android 6

Reply:
updated to `3.3.3`